console/src/components/MapCore/MapCore.less

```diff
@@ -32,6 +32,22 @@
 			text-align: end;
 		}
 	}
+
+	.applySkins({
+		.marker {
+			height: 45px;
+			width: 45px;
+			background-color: @agate-accent;
+			-webkit-mask-image: url('../Dashboard/svg/marker.svg');
```

Comment:
did the marker SVG get committed?

Would you mind putting it somewhere in the `/console/assets` folder with the rest of the images? (They all moved recently.)